client/components/gsuite/gsuite-new-user-list/domains-select.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React, { ChangeEvent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormSelect from 'components/forms/form-select';
+
+interface Props {
+	domains: string[];
+	onChange: ( event: ChangeEvent< HTMLInputElement > ) => void;
+	value: string;
+}
+
+const DomainsSelect = ( { domains, onChange, value }: Props ) => {
```

Comment:
Shouldn't we name this component `GSuiteDomainsSelect`?

Reply:
fixed